app/views/chargeback/_cb_rate_show.html.haml

```diff
@@ -18,23 +18,20 @@
         %th= _('Group')
         %th= _('Description')
         %th= _('Rate')
-        %th= _('Per Time')
-        %th= _('Per Unit')
+        %th= _('Units')
     %tbody
       - @sb[:selected_rate_details].each do |r|
```

Comment:
@jrafanie yes, I fixed it. Now, I obtain the currency code before entering the loop considering that this code is the same for all the chargeback_rate_details.